File: express/service/mongoModel/userModel.js
var mongoose = require("mongoose");

var userSchema = mongoose.Schema(
	{
		id: {
			type: String,
			required: true,
		},
		password: {
			type: String,
			required: true,
		},
		username: {
			type: String,
			required: true,
		},
		email: {
			type: String,
		},
		name: {
			type: String,
		},
		address: {
			type: String,
		},
		dob: {
			type: Date,
		},
		phone: {
			type: Number,
		},
		role: {
			type: String,
		},
	},
	{
		toJSON: {
			transform: function (doc, ret) {
				ret.id = ret._id;
				delete ret._id;
				delete ret.__v;
			},
		},
	}
);

module.exports = userModel = mongoose.model("user", userSchema);

File: flask/service/mongoModel/userModel.py
import datetime
from ..dbMongo import db

class UserModel(db.Document):
	id = db.StringField(primary_key=True)
	password = db.StringField(required=True)
	username = db.StringField(required=True, unique=True)
	email = db.StringField()
	name = db.StringField()
	address = db.StringField()
	dob = db.DateTimeField(default=datetime.datetime.now)
	phone = db.DecimalField()
	role = db.StringField()
